### ISSUE TITLE
Branding toolkit and poster

### ISSUE THREAD

**Issue by maxwofford** [hackclub/OnBoard#143]
We want to have poster designs, so we're working on a new poster design with Alex, a contractor designer. The brand kit is a tool for consistent and quick creation of materials like future posters.

Brand kit includes:
- colors
- assets
- branding with examples

**Comment by kvnyng** [comment]
I followed up with Alex on an update!

![Image](https://github.com/hackclub/OnBoard/assets/45344788/d28eaff7-d955-4c74-841a-264a3445305e)

**Comment by karamassie** [comment]
are these final @maxwofford  ?

**Comment by maxwofford** [comment]
Nope! First draft. We're having trouble getting in touch with Alex, the designer
